config/cmake/HDFCXXCompilerFlags.cmake

```diff
@@ -173,16 +174,16 @@ if (NOT MSVC AND NOT MINGW)
 
     # Append more extra warning flags that only gcc 6.x+ knows about
     if (NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS 6.0)
-      # autotools always add the C flags with the CXX flags
+      # Use the C warnings as CXX warnings are the same
```

Comment:
I feel that this phrase is gramatically incorrect but I don't know what it should be.

Reply:
I'm not sure it's gramatically incorrect, but maybe something like "Use the C warnings for CXX since both use the same warnings flags" is clearer?

Reply:
I feel like Larry's statement is better

Reply:
I agree it isn't grammatically correct, but 1. it is better then the previous, 2. I didn't mean it to be a proper sentence, and 3. It is a minor issue and develop would also need to be fixed.

Reply:
Ok, so let's test this in hdf5_1_12, and if it works we'll update the comments in another set of PRs.